zerver/tests/test_custom_profile_data.py

```diff
@@ -743,7 +743,8 @@ def test_list_order(self) -> None:
             .order_by("-order")
             .values_list("order", flat=True)
         )
-        try_reorder_realm_custom_profile_fields(realm, order)
+        assert all(isinstance(item, int) for item in order)
+        try_reorder_realm_custom_profile_fields(realm, cast(Iterable[int], order))
```

Comment:
This should have a comment pointing at https://github.com/typeddjango/django-stubs/issues/444, and/or we should encourage upstream to fix it, since it looks pretty simple.